app/api-v1/controllers/Order/helpers.js

```diff
@@ -0,0 +1,27 @@
+const db = require('../../../db')
+const { NoToken } = require('../../../utils/errors')
+
+exports.mapOrderData = async (data) => {
+  const records = await db.getRecipeByIDs(data.items)
+  const tokenIds = records.map((el) => el.token_id)
+  if (!tokenIds.every(Boolean)) throw new NoToken('recipes')
+
+  const recipes = data.items.reduce((id, output) => {
+    if (id) {
+      output[id] = {
+        type: 'TOKEN_ID',
+        value: id,
+      }
+    }
+
+    return output
+  }, {})
```

Comment:
The recipe list isn't part of the metadata. It'll be part of the `input` and `output` token sets

Reply:
I thought we were also putting the token ids in the metadata?

Reply:
Got ya! Updated.